ibm_vpd_app.cpp

```diff
@@ -1426,6 +1426,63 @@ void doEnableAllMuxChips(const nlohmann::json& js)
     }
```

Comment:
Can we have little details about why we need this change. and test results, may some couts to show data before and after execution.